bin/viam-mlmodelservice-triton.sh.envsubst

```diff
@@ -14,4 +14,4 @@ exec docker run \
      -v ${DOLLAR}SOCKET_DIR:${DOLLAR}SOCKET_DIR \
      -v ${DOLLAR}VIAM_DIR:${DOLLAR}VIAM_DIR \
      $TAG \
-     sudo -u \#$(id -u) LD_PRELOAD=libjemalloc.so.2 /opt/viam/bin/viam_mlmodelservice_triton "$@" 2>&1
+     sudo -u \#$(id -u) LD_PRELOAD=libjemalloc.so.2 VIAM_MODULE_DATA="$VIAM_MODULE_DATA" /opt/viam/bin/viam_mlmodelservice_triton "$@" 2>&1
```

Comment:
Sneaky trick alert: we need to mention the environment variable here and not in a `-e ...` line around line 15ish because the `-e` would set the environment variable for the command that gets run, which is `sudo`, which then _does not_ forward it on to the actual server! Is that worth explaining in a comment in the code? I'm unsure, maybe.

Reply:
I think a comment is worthwhile, up around line 8 or so.

Meanwhile, I have a question. Is the path that `VIAM_MODULE_DATA` points to specific/unique to the module, or the *instance* of the module? In other words, if I have the triton module installed, and then I create two instances of it, will they run with the same or different value here?

Reply:
It's for the module as a whole, not specific instances of it. You imply a good point, that perhaps each instance of a component/service should get its own. I suspect Netcode owns this stuff; try filing a Jira ticket with them.

Reply:
Yes and no. I can see use cases for both. I actually want to think carefully about the implications for triton, because I eventually want multiple instances of the triton module which share a repository path to share a TRITONSERVER_Server instance, per https://viam.atlassian.net/browse/RSDK-9265. I'm also curious whether it is possible to deploy two different versions of an mlmodel module to the same instance. If so, and those were configured as the `model_path` of two different triton server module instances, would collisions ensue?